docsets/DraftJS/docset.json

```diff
@@ -1,11 +1,27 @@
 {
     "name": "Draft.js",
-    "version": "0.6.0/2016-04-27_20:52:21",
-    "archive": "Draft.js.tgz",
+    "version": "0.10.0",
+    "archive": "DraftJS.tgz",
     "author": {
-        "name": "Remko Tronçon",
-        "link": "https://el-tramo.be"
+        "name": "Michael Sharer",
+        "link": "http://29degre.es"
     },
-    "aliases": ["draft", "draftjs"],
-    "specific_versions": []
+    "aliases": [
+        "Draft",
+        "DraftJS",
+        "React",
+        "WYSIWYG",
+        "RichText",
+        "Editor"
+      ],
+    "specific_versions": [
+        {
+          "version": "0.10.0",
+          "archive": "versions/0.10.0/DraftJS.tgz"
+        },
+        {
+          "version": "0.1.0",
+          "archive": "versions/0.1.0/DraftJS.tgz"
```

Comment:
It looks like specific versions were removed in this commit: https://github.com/Kapeli/Dash-User-Contributions/commit/2d39e33a21e0d50af403c00e6725c52f23d36bd7.

My recommendation would be to just remove the specific version entry for `0.1.0` from docset.json and just have one for `0.10.0`. I can make this change if it's ok.